pkg/detector/library/testdata/fixtures/php.yaml

```diff
@@ -1,4 +1,4 @@
-- bucket: "php::GitHub Security Advisory Composer"
+- bucket: "composer::GitHub Security Advisory Composer"
```

Comment:
I realize this is just a test file but in reality aren't these keys already in use somewhere with the existing prefix of `php`? Wouldn't changing the prefix break that?

Reply:
You're right, but this feature is mainly for our SaaS. They will introduce custom prefix from now, so it doesn't break anything. I don't think OSS community uses this feature, but I'll announce it just in case.

Reply:
Sounds good! Should we play safe and announce first before we merge this PR?

Reply:
If we change the schema of trivy-db, it affects all users even if they don't upgrade trivy version. But this change affects only users who upgrade trivy to the latest version, v0.13.0. In addition, I think nobody uses this feature. It looks like it is enough to announce it in the changelog of v0.13.0. I mean we can merge this PR now, and release the new version with the note in the changelog.